activation/atxdb.go

```diff
@@ -121,16 +132,16 @@ func (db *DB) UnsubscribeAtx(id types.ATXID) {
 
 // ProcessAtxs processes the list of given atxs using ProcessAtx method
 func (db *DB) ProcessAtxs(atxs []*types.ActivationTx) error {
-	seenMinerIds := map[string]struct{}{}
+	// seenMinerIds := map[string]struct{}{}
 	for _, atx := range atxs {
-		minerID := atx.NodeID.Key
+		/* minerID := atx.NodeID.Key
```

Comment:
should we delete this code?

Reply:
No, this is alongside a TODO with blacklisting, I think we should keep it as is for now

Reply:
@antonlerner I wonder why you commented out this code - it just prints an error log when something happens that's not supposed to happen.

If you've seen this happening - we should look into why.

In any case, I think it should be uncommented.